### ISSUE TITLE
Accessibility Phase 2: Dashboards & Panels

### ISSUE THREAD

**Issue by polibb** [grafana/grafana#56609]
# Accessibility in the Realm of OSS Dashboards Squad 

## Ongoing
- Bug hunting sessions bi-weekly. Anyone from the team is welcome to join whenever they can. 
- A11y checks (manual and through browser extensions) on each PR we go through. 

#### To _Automatically_ Find and Log on GitHub 
- [ ] Issues found and logged of the pages we own
  - [ ] Run FastPass on any open dashboard page
  - [ ] Run FastPass on open dashboard page with open Share modal with each submenu opened one by one
  - [ ] Run FastPass on open dashboard page with open Save modal/side view
  - [ ] Run FastPass on open dashboard page with open 'Add panel' choice selection
  - [ ] Run FastPass on open dashboard page with open 'Add panel from panel library' embedded selection
  - [ ] Run FastPass on a dashboard page with >= 2 separate rows of panels
  - [ ] Run FastPass on a dashboard page with open Time Picker
  - [ ] Run FastPass on a dashboard in Cycle view mode
  - [ ] Run FastPass on a dashboard in Kiosk mode
  - [ ] Run FastPass on a dashboard with panels with one 'Share panel' modal open with each submenu opened one by one
  - [ ] Run FastPass on a dashboard with panels with one 'Inspect' of a panel opened: with each submenu opened on by one
  - [ ] Run FastPass on Panel View page
  - [ ] Run FastPass on Panel Edit page
    - Pay special attention to Query runner, Panel Thresholds, Panel datalinks, Panel field overrides, Panel annotations
  - [ ] Run FastPass on Dashboard Settings page **General**
  - [ ] Run FastPass on Dashboard Settings page **Annotations**
  - [ ] Run FastPass on Dashboard Settings page **Annotations** inside **New annotation query** form
  - [ ] Run FastPass on Dashboard Settings page **Annotations** within an existing annotation overview
  - [ ] Run FastPass on Dashboard Settings page **Variables** with and without any Renamed/Missing variables
  - [ ] Run FastPass on Dashboard Settings page **Variables** inside **New** form
  - [ ] Run FastPass on Dashboard Settings page **Variables** within an existing variable
  - [ ] Run FastPass on Dashboard Settings page **Links**
  - [ ] Run FastPass on Dashboard Settings page **Links** inside a **New** form
  - [ ] Run FastPass on Dashboard Settings page **Links** within an existing link
  - [ ] Run FastPass on Dashboard Settings page **Versions**
  - [ ] Run FastPass on Dashboard Settings page **Versions** with a Comparison between 2 versions
  - [ ] Run FastPass on Dashboard Settings page **JSON Model**
  - [ ] Run FastPass on Import Dashboard pages (each step)
  - [ ] Run FastPass on Library Panels page

#### Logged issues
- [ ]

#### To _Manually_ Find and Log on GitHub
- [ ] **TBD**: Accessibility Assessment per scenario of all user journeys possible inside the realm we are responsible for as a squad.
